source/_integrations/velbus.markdown

```diff
@@ -31,18 +31,33 @@ ha_integration_type: hub
 
 The **Velbus** {% term integration %} is used to control [Velbus](https://www.velbus.eu/?lang=en) modules. It supports the Velbus USB, Velbus serial and a TCP/IP gateway.
 
+<div class='note'>
 The pushbutton LEDs of input modules are disabled by default. These can be enabled from the `Devices` panel in the `Configuration` page of the web interface.
```

Comment:
I think we need to move this further down.

Serving a note before the user has even read the configuration documentation, is quite odd :)